components/database/freetds/freetds.p5m

```diff
@@ -26,16 +27,6 @@ file etc/freetds.conf path=usr/share/doc/freetds/freetds.conf.sample
 file etc/locales.conf path=usr/share/doc/freetds/locales.conf.sample
 file etc/pool.conf path=usr/share/doc/freetds/pools.conf.sample
 
-hardlink path=usr/bin/bsqldb target=../lib/isaexec pkg.linted.userland.action002.0=true
```

Comment:
Restore isaexec symlinks or ship 64-bit binaries only

Reply:
64-bit-only now.

Reply:
There's a misunderstanding here - I've spoken about /usr/bin/* files. We still need 32-bit libraries.